children/README.md

```diff
@@ -0,0 +1,156 @@
+# STAC API - Children
+
+- [STAC API - Children](#stac-api---children)
+  - [Link Relations](#link-relations)
+  - [Endpoints](#endpoints)
+  - [Pagination](#pagination)
+  - [Example](#example)
+
+- **OpenAPI specification:** [openapi.yaml](openapi.yaml) ([rendered version](https://api.stacspec.org/v1.0.0-beta.5/children))
+- **Conformance URI:** <https://api.stacspec.org/v1.0.0-beta.5/children>
+- **Extension [Maturity Classification](../extensions.md#extension-maturity):** Pilot
+- **Dependencies**: [STAC API - Core](../core)
+
+A STAC API can return information about all STAC [Catalogs](../stac-spec/catalog-spec/catalog-spec.md) available using a link
```

Comment:
I'm confused if the server is expected to return just the first generation of children, the last generation of children, or the entire ancestry.  For example, if I had Landsat catalogs organized as `/catalogs/landsat_8_c1/{path}_{row}_{date}`, the `/children` endpoint could:
1. Return all `{path}` catalogs (immediate children).
2. Return every combination of `{path}_{row}_{date}` (last generation of children).
3. Return every combination of `{path}`, `{path}_{row}`, and `{path}_{row}_{date}` (entire ancestry).

Reply:
Hmm... is this a conformance class a "simple" way for providers to express their "preferred list of catalogs and collections to show on a frontpage"?

Reply:
To answer @geospatial-jeff 's question, it's up to the implementer. I think typically it will return exactly the same set of children that have `child` relations from the root. The benefit to having it at this endpoint is that the title, desc, etc can all be returned, so a client doesn't have to retrieve every single child link just to find out the title to display -- this was Rob's use case.

Maybe "simple" isn't the right word. I'll think about this.